src/ALE/MOM_ALE.F90

```diff
@@ -952,7 +952,7 @@ subroutine ALE_initRegridding(GV, max_depth, param_file, mod, regridCS, dz )
   call initialize_regridding( GV%ke, coordMode, interpScheme, regridCS, &
                               compressibility_fraction=compress_fraction )
 
-  if (coordMode(1:2) == 'Z*') then
+  if (coordMode(1:5) == 'ZSTAR') then
```

Comment:
OK, yes, I thought it was a stretch changing something so central! The motivation, as you noted, was to keep the '*' symbol out of the config and the diag name suffixes in the diag table. I like your idea, another possibility would be to use 'Z' to replace 'Z*' and 'ZSTAR' everywhere. I'm not enough of an oceanographer to know how important it is to make the 'star' explicit.